godtools/ViewControllers/Platform/ToolDetailViewController.swift

```diff
@@ -42,6 +43,7 @@ class ToolDetailViewController: BaseViewController {
             .joined(separator: ", ")
         
         self.displayButton()
+        self.bannerImageView.image = BannerManager.shared.loadFor(resource!)
```

Comment:
In this case we are just loading from the disk. I don't think we need to use a promise here.